Credits.html

```diff
@@ -41,9 +41,10 @@ <h1>Credits</h1>
     <li><a class=clickable onClick="linkHandler('https://www.deviantart.com/blueamnesiac');">BLUEamnesiac</a>, for the sweet HD icons used on the home screen</li>
     <li>The <a class=clickable onClick="linkHandler('https://discord.gg/qke279EUa8');">WebTV Discord server</a>, for being a great community</li>
     <li style=font-size:1.2rem;>Pink_Sheep, for his ideas</li>
+    <li style=font-size:1.2rem;>SKCro for the original WebTV HD repo on Github</li>
```

Comment:
You made my name smaller than all the other names because you forgot to remove the style attribute. Brilliant.